### PR TITLE
Added Custom namespace support and default values for cloud-drive support for Quickstart

### PR DESCRIPTION
**Pull request by pragrawal-px**
Signed-off-by: pragrawal10 <pragrawal@purestorage.com>

**What this PR does / why we need it**:
This PR adds Custom namespace support to the chart. Also default values in values.yaml is changed to support cloud drives provisioning for container marketplace

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by pragrawal-px** [comment]
TODO:-
1) Solve the CSI=false issue.

File: charts/portworx/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: portworx
-version: 2.11.0
+version: 2.11.2
```
Comment (sharma-tapas) [inline charts/portworx/Chart.yaml]: make this 2.12.0

File: charts/portworx/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 # appVersion: 2.11.2
```
Comment (sharma-tapas) [inline charts/portworx/Chart.yaml]: un-comment this and move to 2.12.0 as value.

File: charts/portworx/values.yaml [hunk 3]
```diff
@@ -49,7 +49,7 @@ miscArgs: none                        # Miscellaneous arguments that will be pas
 disableStorageClass: false            # Instructs Operator to not install the default Portworx StorageClasses.
 
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 2.11.1                        # Optional: version of Stork. For eg: 2.7.0, when it's empty Portworx operator will pick up
+storkVersion: 2.11.2                        # Optional: version of Stork. For eg: 2.7.0, when it's empty Portworx operator will pick up
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: stork version 2.12.0 also change
imageVersion: 2.12.0               
pxOperatorImageVersion: 1.10.0

File: charts/portworx/values.yaml [hunk 4]
```diff
@@ -108,7 +108,7 @@ aws:
   accessKeyId: 
   secretAccessKey: 
 
-deleteType:
+deleteType: UninstallAndWipe
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: Will this install PX or not?

File: charts/portworx/Chart.yaml (hunk 1 above)
Comment (pragrawal-px) [reply charts/portworx/Chart.yaml]: Done

File: charts/portworx/values.yaml (hunk 4 above)
Comment (pragrawal-px) [reply charts/portworx/values.yaml]: This field is different from the deletestrategy field. This value is used for the shell script that patches the STC before deletion and doesn't interfere with installation

File: charts/portworx/Chart.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 name: portworx
-version: 2.11.0
-# appVersion: 2.11.2
+version: 2.12.0
```
Comment (sharma-tapas) [inline charts/portworx/Chart.yaml]: 2.12.2

File: charts/portworx/values.yaml [hunk 6]
```diff
@@ -1,23 +1,26 @@
...
 
+# This is the marketplace subscribed product: PX-ENTERPRISE or PX-ENTERPRISE-DR
+awsProduct: "PX-ENTERPRISE"         #This is the default.
+
 deployOperator: true                    # Deploy the Portworx operator
 deployCluster: true                     # Deploy the Portworx cluster
 
-imageVersion: 2.11.2                  # Version of the PX Image.
-pxOperatorImageVersion: 1.9.0       # Version of the PX operator image.
+imageVersion: 2.12.0                 # Version of the PX Image.
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: 2.12.2

File: charts/portworx/values.yaml [hunk 7]
```diff
@@ -1,23 +1,26 @@
...
+pxOperatorImageVersion: 1.10.0       # Version of the PX operator image.
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: 1.10.3

**Review by sharma-tapas — APPROVED** [review]

File: charts/portworx/templates/storage-cluster.yaml [hunk 8]
```diff
@@ -202,7 +202,7 @@ spec:
         key: AZURE_TENANT_ID
   {{- end }}
   - name: PX_IMAGE
-    value: docker.io/portworx/px-enterprise:2.12.0
+    value: docker.io/portworx/px-enterprise:2.12.2
```
Comment (sharma-tapas) [inline charts/portworx/templates/storage-cluster.yaml]: 2.13.2

File: charts/portworx/values.yaml [hunk 9]
```diff
@@ -6,8 +6,8 @@ awsProduct: "PX-ENTERPRISE"         #This is the default.
 deployOperator: true                    # Deploy the Portworx operator
 deployCluster: true                     # Deploy the Portworx cluster
 
-imageVersion: 2.12.0                 # Version of the PX Image.
-pxOperatorImageVersion: 1.10.0       # Version of the PX operator image.
+imageVersion: 2.12.2                 # Version of the PX Image.
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: 2.13.2

File: charts/portworx/values.yaml [hunk 10]
```diff
@@ -6,8 +6,8 @@ awsProduct: "PX-ENTERPRISE"         #This is the default.
...
+pxOperatorImageVersion: 1.10.3       # Version of the PX operator image.
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: 23.3.0

File: charts/portworx/values.yaml [hunk 11]
```diff
@@ -52,7 +52,7 @@ miscArgs: none                        # Miscellaneous arguments that will be pas
 disableStorageClass: false            # Instructs Operator to not install the default Portworx StorageClasses.
 
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 2.12.0                        # Optional: version of Stork. For eg: 2.7.0, when it's empty Portworx operator will pick up
+storkVersion: 2.12.2                        # Optional: version of Stork. For eg: 2.7.0, when it's empty Portworx operator will pick up
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: 23.2.0

File: charts/portworx/values.yaml [hunk 12]
```diff
@@ -124,5 +124,5 @@ deleteStrategy:                           # Optional: Delete strategy for the po
   type:                                   # Valid values: Uninstall, UninstallAndWipe
 
 versions:
-  enterprise: "2.12.0"
+  enterprise: "2.12.2"
```
Comment (sharma-tapas) [inline charts/portworx/values.yaml]: 2.13.2

**Comment by pragrawal-px** [comment]
@sharma-tapas Somehow you reviewed the outdated version where the charts have 2.12 px version.
in the latest changes I have also added 2.13.1 and 2.13.2 charts

**Pull request merged**